lib/ansible/modules/packaging/os/yum.py

```diff
@@ -687,6 +690,8 @@ def install(module, items, repoq, yum_basecmd, conf_file, en_repos, dis_repos, i
             # download package so that we can check if it's already installed
             package = fetch_rpm_from_url(spec, module=module)
             nvra = local_nvra(module, package)
```

Comment:
There are more `local_nvra` calls in this file (please rebase, some were added since this PR), should we check for the return values of those too?